source/MaterialXCore/Util.cpp

```diff
@@ -62,6 +62,14 @@ bool isValidNamespace(const string& name)
     return isValidName(name) && name.find(':') == std::string::npos;
 }
 
+bool isValidEnvironmentPath(const string& path)
```

Comment:
This belongs more as part of the Format module as a file utility.
- Also I don't see anything specific to "environment" but it's more a path
checker.
- Finally there are existing constants for path separators you can use.

Reply:
The naming was intended to point out that we are checking the Preference Environment Variables. I can improve the wording, or just leave isValidPath.